src/gax-internal/src/lib.rs

```diff
@@ -12,31 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Implementation details for Google Cloud clients using HTTP as their
-//! transport.
+//! Implementation details for Google Cloud clients.
 //!
 //! All the types, traits, and functions defined in this crate are **not**
 //! intended for general use. This crate will remain unstable for the
 //! foreseeable future, even if used in the implementation for stable client
 //! libraries. We (the Google Cloud Client Libraries for Rust team) control
 //! both and will change both if needed.
+//!
+//! The types, traits, and functions defined in this crate are undocumented.
+//! This is intentional, as they are not intended for general use and will be
+//! changed without notice.
 
-use auth::credentials::{create_access_token_credential, Credential};
-use gax::backoff_policy::BackoffPolicy;
-use gax::error::Error;
-use gax::error::HttpError;
-use gax::error::ServiceError;
-use gax::exponential_backoff::ExponentialBackoff;
-use gax::loop_state::LoopState;
-use gax::options;
+// TODO(#1539) - remove these once all the generated clients use gax:: directly.
 pub use gax::options::ClientConfig;
```

Comment:
Ah, you have a plan.

Reply:
yes, and I can execute on it even sooner than I thought.